portfolio/src/main/webapp/script.js

```diff
@@ -144,11 +144,20 @@ function initMap() {
     animation: google.maps.Animation.DROP,
     map: map
     });
-    
     marker.addListener('click', () => {
       map.setZoom(11);
       map.setCenter(marker.getPosition());
     });
+    let infowindow = new google.maps.InfoWindow({
```

Comment:
I would look into using the forEach function, instead of the for-loop you have defined above, I think itll make this a bit more readable
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach

Reply:
got it! Fixed